cms/services.ts

```diff
@@ -116,6 +116,7 @@ export type ServiceDetail = Pick<
   | 'reviews'
   | 'format'
   | 'shortDescription'
+  | 'provider'
```

Comment:
This is the type for the data on each service's page, I don't think we need the provider there at the moment.